lib/components/base/Pagination.vue

```diff
@@ -28,7 +28,7 @@
           'page-number current': page === item,
           shrink: item > 99,
         }"
-        :href="linkFunction(item)"
+        :href="linkFunction?.(item)"
```

Comment:
```suggestion
        :href="linkFunction(item)"
```

Optional chaining can be avoided by following the suggestion below.